e2e/sedge/monitoring_stack_test.go

```diff
@@ -16,14 +16,22 @@ limitations under the License.
 package e2e
 
 import (
+	"runtime"
 	"testing"
 
 	base "github.com/NethermindEth/sedge/e2e"
 	"github.com/stretchr/testify/assert"
 )
 
+func skipIfNotAMD64(t *testing.T) {
+	if runtime.GOARCH != "amd64" {
+		t.Skip("Skipping test on non-AMD64 architecture")
+	}
+}
+
```

Comment:
Should we then remove `-skip TestE2E_MonitoringStack` from [here](https://github.com/NethermindEth/sedge/blob/4371d71a6be4fc18cef5ac487df0cb158e75fcb8/Makefile#L59C1-L60C78)? I think we could remove `make e2e-test-windows` as a whole.